tools/swarm/src/cli.rs

```diff
@@ -10,6 +10,12 @@ pub struct Cli {
     /// The Unicode `seed` string for deterministic key-generation.
     #[arg(long, short)]
     pub seed: Option<String>,
+    /// Explicit keypair in json format. Takes precedence over seed parameter.
+    #[arg(long)]
+    pub key_pair: Option<String>,
+    /// Genesis signature. Should be set with --key-pair.
+    #[arg(long, requires = "key_pair")]
+    pub signature: Option<String>,
```

Comment:
Why?

Swarm only needs to support `--genesis-signature` argument, and simply pass it down to the generated `yaml` in a form of `GENESIS_SIGNATURE` env.